context/php/php-fpm.conf

```diff
@@ -0,0 +1,5 @@
+[global]
+pid = /var/run/php7.2-fpm.pid
```

Comment:
It seems this depends on PHP version. Should we separate the files, or replace with argument?

Reply:
Removed version from PID file path